interface/xtreemfs/OSD.proto

```diff
@@ -354,8 +354,19 @@ message xtreemfs_xloc_set_invalidateRequest {
 }
 
 message xtreemfs_xloc_set_invalidateResponse {
-  required bool is_primary = 1;
-  optional ReplicaStatus status = 2;
+  enum LeaseState {
```

Comment:
GlobalTypes seems to be the right place for this enum. Could we combine ReplicaState and LeaseState to a common enum? If not, LeaseState is fine for me as the meaning of the states is obvious.